test/Sentry.Tests/Integrations/NetFxInstallationsIntegrationTests.cs

```diff
@@ -0,0 +1,47 @@
+#if NETFX
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Sentry.Integrations;
+using Sentry.PlatformAbstractions;
+using Xunit;
+
+namespace Sentry.Tests.Integrations
+{
+    public class NetFxInstallationsIntegrationTests
+    {
+        [SkippableFact]
+        public void Register_IgnoredIfPlattformIsMono()
+        {
+            Skip.If(!Runtime.Current.IsMono());
```

Comment:
Don't u need this off in order to test this?
```suggestion
```

Reply:
The test just checks when you are running mono that the event processor isn't added.

`public void Register_PlattformIsMono_NetFxInstallationsEventProcessorNotAdded()`
I suppose the function name caused some confusion.

Reply:
or Register_CurrentRuntimeIsMono_NetFxInstallationsEventProcessorNotAdded